pkgs/development/haskell-modules/configuration-hackage2nix.yaml

```diff
@@ -69,6 +69,8 @@ core-packages:
 default-package-overrides:
   # pandoc-2.9 does not accept the 0.3 version yet
   - doclayout < 0.3
+  # gtk 4 is still under development
+  - gi-gdkx11 < 4
```

Comment:
Let me see if I understand this.

`gi-gtk` that supports gtk-4 is still not released to hackage, so it doesn't make any sense to track the latest version of `gi-gdk` (and `gi-gdkx11`)?

Could you expand to this comment with information about when you expect will be able to remove this override?  (Maybe when `gi-gtk-4` is released to hackage?)

Reply:
`gtk4` isn't even in nixpkgs yet for that matter, it's still in development.